Target-MVD-Mathias/Extensions/UIColorExtension.swift

```diff
@@ -19,5 +19,7 @@ extension UIColor {
   static let macaroniAndCheese = UIColor(r: 239, g: 198, b: 56)
   static let transparentWhite = UIColor(r: 255, g: 255, b: 255, alpha: 0.7)
   static let brightSkyBlue = UIColor(r: 47, g: 188, b: 247)
-  
+  static let white70 = UIColor(r: 244, g: 244, b: 244, alpha: 0.7)
```

Comment:
Maybe you can add some "customization" to these colors. For example: 
`tWhite70`, so you know that the color is from Target. WDYT?

Reply:
I think that's a good idea, I'll do it.